cypress/integration/E2E_tests/MessagingTests/messaging.spec.js

```diff
@@ -21,26 +21,30 @@ describe("On the messaging page, can", () => {
     cy.get("[data-cy=search]")
       .click()
       .type("Jane Doe")
-      .type('{enter}');
+      .type("{enter}");
   });
   it("create new chat", () => {
-    cy.get("[data-cy=new-message]").click();
+    cy.get("[data-cy=new-message]").click({ force: true });
+    cy.get("[data-cy=chat-user]").click({ force: true }).type("John").type("{enter}")
+    cy.get("[data-cy=list-item]").first().click()
+    cy.get("[data-cy=next-button]").click()
+    cy.get("[data-cy=dialog]").should("not.exist")
```

Comment:
Looks like the only thing we're testing for here is that no dialog pops up. Is there anything on the newly created conversation page we can look for that confirms it was actually created?

Reply:
For the moment, no conversation page is created, actually nothing happens after "next" is pressed, I could add code that is added to the console once the "next" button is pressed and check that (I assume you can do that in Cypress).

Reply:
I think it would be better to check for an affirmative rather than a negative. Even if for now there's just a dialog that pops up that says "new conversation created" or something like that and then test that it says that.

Reply:
Added an alert when the button is created, and added the cypress test!